products/firefox_iOS/review_iOSstrings.md

```diff
@@ -0,0 +1,75 @@
+#Reviewing strings for a new release of Firefox for iOS
+Usually, a new release of Firefox for iOS means an update to strings. If this is the case, a pull request will be made by the iOS team, in order to land these new strings in the [mozilla-l10n firefoxios-l10n repository](https://github.com/mozilla-l10n/firefoxios-l10n).
+
+At this point, the PR is reviewed by an l10n-driver - most often by the PM in charge of the project.
+
+Let's go over some of the steps needed over time in order to review correctly strings for a new release.
+
+##Reviewing the PR
+When you're getting close to the end of the [Firefox for iOS l10n cycle](https://wiki.mozilla.org/Firefox_for_iOS_Train_Schedule), the iOS team will make a pull request in order to land new strings for the upcoming release.
+
+Let's consider a past PR for this:
+https://github.com/mozilla-l10n/firefoxios-l10n/pull/19
+
+The first thing you'll want to do is to check the changes to the /templates folder:
+https://github.com/mozilla-l10n/firefoxios-l10n/pull/19/commits/8a8ecbcb748b0f023557d6c1b07e6208a2e953ea
+
+This folder is used by Pootle, and it's the easiest point to start from, because it doesn't have translations to make the diff larger and harder to read.
+
+###Acceptable changes
+Changes to attributes like tool-version and build-num are expected, they happen every time you change the version of Xcode.
+
+Same for the utf-8 -> UTF-8 at the beginning, and version number bumping to 6.0. Note that these should now be automatically translated (so will not appear as new strings for localizers and they will not need to touch them).
+
+Changes to comments (<note>) are irrelevant in terms of string updates, so they're also OK.
+
+Note also that the script converts Unicode literals into UTF-8 characters. Example: https://github.com/mozilla-l10n/firefoxios-l10n/pull/19/commits/713def82602224556d797b46e3f6d611297ee3e3#diff-42d28e544c7a30099950b7a09a69f2caL2289
+
+Note that the "&#173;" was not lost - it was converted to the invisible UTF-8 character.
+
+###Potentially problematic changes (removals)
+Then you start scrolling down, checking if the new strings make sense: the first one you find is NSCameraUsageDescription.
+
+You mostly need to pay attention to the strings removed. Double-check each time that iOS team clearly realizes that we won't be able to release updates to v5.x once we export the strings.
```

Comment:
Don't use we: either l10n-drivers, or passive form, e.g. "it won't be possible", "strings are exported".

Replace v5.x with a more generic previous version.